lib/contextual_logger.rb

```diff
@@ -136,7 +136,7 @@ def format_message(severity, timestamp, progname, message, context: {})
       if @formatter
         @formatter.call(severity, timestamp, progname, { message: ContextualLogger.normalize_message(message), **context })
       else
-        "#{basic_message_formatting(severity, timestamp, progname, message, context: context).to_json}\n"
+        "#{basic_message_formatting(severity, timestamp, progname, message, context: context)}\n"
```

Comment:
Can  you get `json` into the method name somewhere? That's really an important part of the contract. And `message` probably isn't important.

Reply:
good call, changed to `basic_json_log_entry` to be more descriptive.